src/System.Security.Cryptography.Encoding/tests/Asn1/Serializer/SimpleDeserialize.cs

```diff
@@ -176,19 +176,19 @@ public static void ReadDirectoryString()
             const string BmpInputHex = "1E0400480069";
             const string Utf8InputHex = "0C024869";
 
-            var ds1 = AsnSerializer.Deserialize<DirectoryString>(
+            var ds1 = AsnSerializer.Deserialize<DirectoryStringAsn>(
                 BmpInputHex.HexToByteArray(),
                 AsnEncodingRules.DER);
 
-            var ds2 = AsnSerializer.Deserialize<DirectoryString>(
+            var ds2 = AsnSerializer.Deserialize<DirectoryStringAsn>(
                 Utf8InputHex.HexToByteArray(),
                 AsnEncodingRules.DER);
 
             Assert.NotNull(ds1);
             Assert.NotNull(ds2);
             Assert.Null(ds1.Utf8String);
-            Assert.Null(ds2.BmpString);
-            Assert.Equal("Hi", ds1.BmpString);
+            Assert.Null(ds2.BMPString);
```

Comment:
this change doesn't match FDG conventions - https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions.  It's also inconsistent with the ASN1.UTF8String property (Utf8String) above now

Reply:
I only aligned it to the class that was already used in the Pkcs code. That said, I am more than happy to change it back and update the Pkcs code instead.

Reply:
The DirectoryName field names in the ASN.1 are `utf8String` and `bmpString`, so the correct names are `Utf8String` and `BmpString` for the fields (and `UTF8String` / `BMPString` for the attributes, since those are the ASN.1 type names).  So, yeah, I guess fix my bad casing in the Pkcs one.